apps/prairielearn/src/components/QuestionsTable.html.ts

```diff
@@ -129,6 +131,22 @@ export function QuestionsTable({
             >
               Tags
             </th>
+            ${showSharingSets
+              ? html` <th
+                  data-field="sharing_sets"
+                  data-sortable="false"
```

Comment:
The PR title explicitly mentions sorting, but this is disabled. I agree with keeping it disabled (it doesn't make sense to sort by a list of items), I just want to make sure you are aware of it. I imagine that you mean "filtering" in the PR title?

Reply:
ya, you're right. I have fixed it in the PR title and description